homeassistant/components/keymitt_ble/translations/en.json

```diff
@@ -0,0 +1,37 @@
+{
+  "config": {
+    "flow_title": "{name} ({address})",
```

Comment:
I added a new name helper to yalexs_ble that could be helpful here.

On macOS the Bluetooth address is a uuid so it might be a bit long to have the full address in the title